src/asciidoctor-kroki.js

```diff
@@ -176,7 +176,8 @@ function diagramBlockMacro (name, context) {
       const diagramType = name
       try {
         const diagramText = vfs.read(target)
-        return processKroki(this, parent, attrs, diagramType, diagramText, context)
+        const baseDir = target.substr(0, target.lastIndexOf('/') + 1)
```

Comment:
I think we should use the built-in base directory from Asciidoctor no?

```js
const baseDir = doc.getBaseDir()
```

Reply:
Done